godel_msgs/srv/TrajectoryPlanning.srv

```diff
@@ -1,13 +1,21 @@
 # The cartesian points representing path to be blended
 ProcessPath path
 
-#extra parameters that tell descartes how to behave
+## extra parameters that tell descartes how to behave
+# frames
 string group_name
 string tool_frame
 string world_frame
+
+# Angle
+bool free_z_rotation   
 float64 angle_discretization
-float64 interpoint_delay
+float64 z_angle_offset
```

Comment:
It's not, and gets removed later. That said, in the interest of this repository, I think it best to make sure it's left in a reasonable state while we go through the process on many commits.

That said, +1.